# 8518-06-06

Error message during extraction:

```shell
Exception in Tkinter callback
Traceback (most recent call last):
  File "c:\users\sam.whindleton\appdata\local\continuum\anaconda3\Lib\tkinter\__init__.py", line 1705, in __call__
    return self.func(*args)
  File "extract_build\extraction_pptx.py", line 205, in get_file
    read_slide(filename_path)
  File "extract_build\extraction_pptx.py", line 876, in read_slide
    basic_info, basic_vid_qa)
  File "extract_build\extraction_out.py", line 992, in excel_data_out
    row, 9, all_act_times[i], ext_sht_content_line)
IndexError: list index out of range
```
* Possible error in first slide containing lesson info.
